### PR TITLE
Add basic header styling

### PR DESCRIPTION
**Pull request by alexabbott**

### CODE REVIEW

**Comment by woodseowl** [comment]
@johnhwhite @ModulesUnraveled Can you two review this pull request?

File: themes/contrib/emulsify/components/_patterns/03-organisms/site/site-header/_site-header.scss [hunk 1]
```diff
@@ -49,10 +88,31 @@
...
+            @extend %transition;
             display: none;
 
             @include medium {
                 display: block;
+                transition-property: margin;
+            }
+
+            &.is-active {
+                background: rgba(1,49,88,0.8);
```
Comment (ModulesUnraveled) [inline themes/contrib/emulsify/components/_patterns/03-organisms/site/site-header/_site-header.scss]: @alexabbott Is this a color that's already in our variables file? If so, you can do something like `rgba($navy, 0.8)`. If not, please add it to the colors file, and use the new variable here.

**Comment by ModulesUnraveled** [comment]
@alexabbott Added one comment, if you can address that, and then check the commit's I've added, I'd like to get this merged so that I can use it in another PR.

Thanks!

**Comment by johnhwhite** [comment]
I confirmed it's $navy and made @ModulesUnraveled 's suggested change.

**Pull request merged**